src/runtime/components/nuxt-icon.vue

```diff
@@ -1,38 +1,81 @@
 <template>
-  <span class="nuxt-icon" :class="{ 'nuxt-icon--fill': !fill }" v-html="icons[name]" />
+  <div class="nuxt-icon-container">
+    <div
+      class="nuxt-icon"
+      :class="{ 'nuxt-icon--fill': fill, 'nuxt-icon--stroke': stroke }"
+      :style="iconSize"
+      v-html="icons[name]"
+    />
+  </div>
```

Comment:
Why use a flex box for an icon importer? More so, I think I think it's nice to have a width and height props while you could remove the stroke props and add the `v-bind="$attrs"` to the line 3-8 div (removing the line 2 div) so you can pass class directly to the icon too 
If I'm not mistaken right now you need to wrap `<nuxt-icon />` inside another element to pass some classes which is quite annoying